Extension/package.nls.json

```diff
@@ -43,6 +43,13 @@
     "c_cpp.configuration.codeAnalysis.excludeBoolean.markdownDescription": { "message": "The glob pattern to match file paths against. Set to `true` or `false` to enable or disable the pattern.", "comment": [ "Markdown text between `` should not be translated or localized (they represent literal text) and the capitalization, spacing, and punctuation (including the ``) should not be altered." ] },
     "c_cpp.configuration.codeAnalysis.excludeWhen.markdownDescription": { "message": "Additional check on the siblings of a matching file. Use `$(basename)` as variable for the matching file name.", "comment": [ "Markdown text between `` should not be translated or localized (they represent literal text) and the capitalization, spacing, and punctuation (including the ``) should not be altered." ] },
     "c_cpp.configuration.codeAnalysis.runAutomatically.markdownDescription": { "message":  "If `true`, code analysis will run automatically on a file after it is opened or saved.", "comment": [ "Markdown text between `` should not be translated or localized (they represent literal text) and the capitalization, spacing, and punctuation (including the ``) should not be altered." ] },
+    "c_cpp.configuration.codeAnalysis.clangTidy.codeAction.showDisable.markdownDescription": { "message": "If `true`, the 'Disable' quick fix will be shown when available (the next time code analysis is run, if this setting was disabled). When used, it adds the warning code to the `C_Cpp.codeAnalysis.clangTidy.checks.disabled` setting.", "comment": [ "Markdown text between `` should not be translated or localized (they represent literal text) and the capitalization, spacing, and punctuation (including the ``) should not be altered." ] },
```

Comment:
It's hard to understand the first sentence because there is "`Disable` quick fix" and "(the next time code analysis is run, if this setting was disabled)". To which does the first `Disable` refer to, could this be more clear if the actual setting ID was used instead of `Disable` quick fix?

For the second disable part, does this mean, the setting `c_cpp.configuration.codeAnalysis.clangTidy.codeAction.showDisable` will be active the next time code analysis is run if `c_cpp.configuration.codeAnalysis.clangTidy.codeAction.showDisable` was previously disabled?

Reply:
Yeah, I'm confused by it too. The option is for showing the "Disable ..." code action, but it requires that code analysis be re-run before the setting takes effect, and it would only take effect if the setting is set to true. And when invoked it'll modify the "*.disabled" setting mentioned. I'll re-word the description.

Reply:
Done